src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -201,7 +219,7 @@ public void parse_invalidValueFollowedByValidValue_success() {
     @Test
     public void parse_resetTags_success() {
```

Comment:
Quick check - should this idea of "reset" be extended to all other fields also? Can you test that they reset correctly, if a user wanted to?

Reply:
i think we could consider resetting description, but that would mean that we will have to allow the user to type in `/nd 
   ` (effectively an empty description, which we currently do not allow).

Reply:
Good point. I think it's your call. Perhaps we can discuss more about it when we think more about CLI optimizations.